tests/TestCase/Routing/Route/RedirectRouteTest.php

```diff
@@ -72,9 +72,9 @@ public function testParseMiss()
     public function testParseSimple()
     {
         $this->expectException(RedirectException::class);
-        $this->expectExceptionMessage('http://localhost/posts');
+        $this->expectExceptionMessage('http://localhost/Posts');
```

Comment:
Why would the URL change too?

Reply:
I was thinking a different route was default.

Reply:
[DashedRoute.php](https://github.com/cakephp/cakephp/blob/master/src/Routing/Route/DashedRoute.php) isn't loaded.

Reply:
Or any inflecting route for that matter :)